### ISSUE TITLE
New rule: "no-duplicate-dependent-keys"

### ISSUE THREAD

**Issue by Turbo87** [ember-cli/eslint-plugin-ember#59]
this should warn for e.g.:

```js
{
  foo: computed('model.foo', 'model.bar', 'model.baz', 'model.foo', function() {
    // ...
  }
}
```

**Comment by michalsnik** [comment]
Good idea @Turbo87

Do you also think about detecting things like:
```
{
  foo: computed('model.{foo,bar}', 'model.bar', function() {
    // ...
  }),
}
```

Or a simple check if any string is duplicated?

**Comment by Turbo87** [comment]
I think having that work too would be awesome although the case that I had in a client project was just the simple one described in my snippet above. It seems that implementing the one in you snippet should not be too hard either.

**Comment by jbandura** [comment]
Picking this up.
